rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -79,28 +79,20 @@ namespace
 // the old string into it. If this becomes a performance problem, we could consider
 // modifying the topic_name in place. But this means we need to be much more
 // careful about who owns the string.
-z_owned_keyexpr_t ros_topic_name_to_zenoh_key(const char * const topic_name, size_t domain_id)
+z_owned_keyexpr_t ros_topic_name_to_zenoh_key(const char * topic_name, size_t domain_id)
```

Comment:
The original `const` qualification seem right to me. We don't want to modify the contents of the original pointer. Is there a reason this change?

Reply:
I changed this for my earlier implementation but not required for current implementation. I made new commit.